Packs/FeedZeroDayLiveTIFusion/pack_metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Zero Day Live TI FUSION Feed",
+    "description": "'Zero Day Live is Blackwired’s flagship product that delivers proprietary",
```

Comment:
please complete the sentence, from their website: `Providing proprietary, holistic, high confidence and precision intelligence data points on Adversaries’ malicious intent.`
or change to `"Indicators feed from a ____"`